aws/cloudwatch_metric.go

```diff
@@ -177,6 +166,15 @@ func listCWMetricStatistics(ctx context.Context, d *plugin.QueryData, granularit
 		},
 	}
 
+	if dimensionName != "" && dimensionValue != "" {
+		params.Dimensions = []*cloudwatch.Dimension{
+			{
+				Name:  aws.String(dimensionName),
+				Value: aws.String(dimensionValue),
+			},
+		}
+	}
+
```

Comment:
@bigdatasourav Is this check required?
Since, it is not using any quals to take input.